_data/versions.json

```
@@ -1,6 +1,7 @@
 {
-  "current": "1.3",
+  "current": "1.7",
```

Still 1.3 here. This only goes up to minor.

```suggestion
  "current": "1.3",
```